examples/typescript-app/src/App.tsx

```diff
@@ -47,7 +46,7 @@ const App = () => (
       <Window>
         <ChannelHeader />
         <MessageList />
-        <MessageInput Input={MessageInputFlat} focus />
+        <MessageInput focus />
```

Comment:
did you mean to keep these changes to App.tsx in there?

Reply:
yep since it's the default don't need it in there explicitly